apollo-sample/build.gradle

```diff
@@ -1,6 +1,17 @@
 buildscript {
   dependencies {
+    // The output JARs don't include transitive dependencies, so we have
+    // to add them manually ourselves :(
+    classpath dep.gradleNodePlugin
     classpath dep.androidPlugin
+    classpath dep.javaPoet
+    classpath dep.kotlinStdLib
+    classpath dep.moshi
+    classpath dep.pluralizer
+    classpath dep.jsr305
+    classpath files(project(':apollo-api').jar.outputs.files.getFiles())
+    classpath files(project(':apollo-compiler').jar.outputs.files.getFiles())
+    classpath files(project(':apollo-gradle-plugin').jar.outputs.files.getFiles())
```

Comment:
this doens't work in CI and causes weird problems. I'll just build a fat jar for now and use that instead until we get artifacts on sonatype snapshots